lib/src/page_list_viewport_gestures.dart

```diff
@@ -415,23 +623,95 @@ class DeprecatedPanAndScaleVelocityTracker {
       }
     }
 
-    if (_previousGesturePointerCount! > 1) {
+    if (_previousGesturePointerCount > 1) {
       // The user was scaling. Now the user is panning. We don't want scale
-      // gestures to contribute momentum, so we set the launch velocity to zero.
+      // gestures to contribute ballistic, so we set the launch velocity to zero.
```

Comment:
I don't understand the user of the word "ballistic" in these two lines. The current use is grammatically incorrect so I'm not sure what you intended.